packages/labs/signals/src/lib/html-tag.ts

```diff
@@ -0,0 +1,46 @@
+/**
+ * @license
+ * Copyright 2023 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+import {
+  html as coreHtml,
+  svg as coreSvg,
+  type TemplateResult,
+} from 'lit/html.js';
+
+import {watch} from './watch.js';
+import {Signal} from 'signal-polyfill';
+
+/**
+ * Wraps a lit-html template tag function (`html` or `svg`) to add support for
+ * automatically wrapping Signal instances in the `watch()` directive.
+ */
+export const withWatch =
+  (coreTag: typeof coreHtml | typeof coreSvg) =>
+  (strings: TemplateStringsArray, ...values: unknown[]): TemplateResult => {
+    // TODO (justinfagnani): use an alternative to instanceof when
+    // one is available. See https://github.com/preactjs/signals/issues/402
```

Comment:
Oh, preact signals added a method to do this. Is there anything in standard signals land?

Reply:
https://github.com/proposal-signals/signal-polyfill/blob/main/src/wrapper.ts#L45

Reply:
That's not exported anywhere. There's an umbrella issue for that type of thing here: https://github.com/tc39/proposal-signals/issues/32